common/scala/src/main/scala/whisk/core/entity/AuthKey.scala

```diff
@@ -16,6 +16,13 @@
 
 package whisk.core.entity
 
+import scala.util.Try
+
+import spray.json.JsString
+import spray.json.JsValue
+import spray.json.RootJsonFormat
+import spray.json.deserializationError
```

Comment:
The explicit imports are a function of the IDE's organize import facility (used for example to import new dependencies or remove unused ones). Is there a way to configure it to use wildcards when the number of imports from a package or class exceeds a threshold? Otherwise this will turn into another edit war.